spec/sections/standard.adoc

```diff
@@ -401,32 +408,34 @@ MUST be treated as true.
     "files": [ // <1>
         {
             "name": "INPUT_FILE", // <2>
-            "mediaType": [ "image/x-hdf5-image" ], // <3>
+            "mediaTypes": [ "image/x-hdf5-image" ], // <3>
             "multiple": false, // <4>
-            "required": true // <5>
+            "partial": true, // <5>
+            "required": true // <6>
         },
         ...
     ]
-    "json": [ // <6>
+    "json": [ // <7>
         {
-            "name": "INPUT_STRING",<7>
-            "type": "string", // <8>
-            "required": false // <9>
+            "name": "INPUT_STRING",<8>
+            "type": "string", // <9>
+            "required": false // <10>
         }
     ]
 }
 ----
 a|
 <1> Optional array containing elements defined by Files Elements sub-section.
 <2> Required string containing name used to inject data via environment variables.
-<3> Required array containing a list of accepted Media types.
+<3> Optional array containing a list of accepted Media types. Omission will disable executor† validation.
```

Comment:
Media should be lower case. I would take out `Omission will disable executor† validation.`

Reply:
+